utils/mock-data-loader/bss/toJson.js

```diff
@@ -1,5 +1,5 @@
 const mocks = require('./index');
 
-const deal = JSON.stringify(mocks.DEALS[0], null, 2);
+const deal = JSON.stringify(mocks.DEALS.dealsNotToSubmit[0], null, 2);
```

Comment:
What even is this file -- Is it used anywhere?

Reply:
Have deleted - not sure what it does but can't see any uses of it